salt/metalk8s/addons/dex/deployed/chart.sls

```diff
@@ -0,0 +1,139 @@
+#!jinja | kubernetes kubeconfig=/etc/kubernetes/admin.conf&context=kubernetes-admin@kubernetes
+{%- from "metalk8s/repo/macro.sls" import build_image_name with context %}
+
+{% raw %}
+
+apiVersion: v1
+kind: Secret
+metadata:
+  labels: {app.kubernetes.io/instance: dex, app.kubernetes.io/managed-by: salt, app.kubernetes.io/name: dex,
```

Comment:
This display of YAML is unreadable for me, I'd rather use block style consistently. See: https://github.com/scality/metalk8s/pull/2024/commits/3cdd70704250346fc938d778311114414171543b